caramel/models.py

```diff
@@ -3,8 +3,8 @@
 
 import sqlalchemy as _sa
 from sqlalchemy.ext.declarative import (
-    declarative_base as _declarative_base,
     declared_attr as _declared_attr,
+    as_declarative as as_declarative,
```

Comment:
I think this will cause issues in the future with pylint or flake8.
Better to formulate it  as:

from  from sqlalchemy.ext.declarative import  ( declared_attr,  as_declarative) 

And just remove the _ from declared_attr below.

Reply:
Also. This change ("sqlalchemy.ext.declarative") is not matched in setup.py below, the versions don't match, and SQLAlchemy 1.1 will fail here.  So the dep in setup.py should be bumped to `>= 1.4`

Reply:
SQLalchemys changelog mentions sqlalchemy.ext.declarative in an example snippet in the [1.1 changelog](https://docs.sqlalchemy.org/en/13/changelog/migration_11.html) so I don't think this should be a problem.